One Trick Companion/Model/Leaderboard.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Leaderboard.swift
+//  One Trick Companion
+//
+//  Created by adam smith on 3/12/22.
+//
+
+import Foundation
+
+//MARK: - RIOT API
+
```

Comment:
spacing

Reply:
what about spacing

Reply:
He means to remove the space from the bottom of the mark and the Struct